examples/3d-tiles/tileset-3d-layer.js

```diff
@@ -203,43 +249,55 @@ export default class Tileset3DLayer extends CompositeLayer {
           normals: {value: positions, size: 3},
           length: positions.length / 3
         },
-        ...coordinateProps,
         numInstances: pointsCount,
-        getPosition: (object, {index, data, target}) => {
-          target[0] = data.positions[index * 3];
-          target[1] = data.positions[index * 3 + 1];
-          target[2] = data.positions[index * 3 + 2];
-
-          if (transform && coordinateProps.coordinateSystem === COORDINATE_SYSTEM.LNGLAT) {
-            target = new Matrix4().multiplyRight(transform).transformVector3(target);
-            Ellipsoid.WGS84.cartesianToCartographic(target, target);
-          }
-
-          return target;
-        },
-        getColor: (...args) => {
-          return this._getColor(tileHeader, ...args);
-        },
-        getNormal: normals
-          ? (object, {index, data, target}) => {
-              target[0] = data.normals[index * 3];
-              target[1] = data.normals[index * 3 + 1];
-              target[2] = data.normals[index * 3 + 2];
-              return target;
+        ...positionProps,
+        getColor: colors
+          ? (...args) => {
```

Comment:
I suggest pre-processing the colors on load so that we don't need an accessor here.